charts/subgraph-radio/values.yaml

```diff
@@ -63,9 +63,11 @@ env:
   REGISTRY_SUBGRAPH: "https://api.thegraph.com/subgraphs/name/hopeyen/graphcast-registry-mainnet"
   # -- Subgraph endpoint to The Graph network subgraph
   NETWORK_SUBGRAPH: "https://api.thegraph.com/subgraphs/name/graphprotocol/graph-network-mainnet"
-  # -- Comma separated static list of content topics to subscribe to
+  # -- Supported Graphcast networks: mainnet, testnet
   GRAPHCAST_NETWORK: "mainnet"
+  # -- Comma separated static list of content topics to subscribe to
   RUST_LOG: "info,hyper=off,graphcast_sdk=info,waku_bindings=off,subgraph_radio=info"
+  # -- For more details of other available env variables you can set, check https://docs.graphops.xyz/graphcast/radios/subgraph-radio#advanced-configuration
 
```

Comment:
@hopeyen added a link to docs for how to set more env vars such as `TOPICS`, `PERSISTENCE_FILE_PATH` or `SLACK_TOKEN`. I don't think those should be added here because only strictly required env vars should be included here - if we set everything else that would force the user to have to provide values to override whatever defaults we set for every env var supported. For example we initially discussed that a `SLACK_TOKEN` and channel are not required to run `subgraph-radio` so you can't really provide a sane default and that would break things. Whereas smth like `INDEXER_ADDRESS` is required for the chart to work so we do expect the user to provide that.